content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst

```diff
@@ -129,10 +136,163 @@ the previous provider.
+
+.. image:: electronic_invoicing/peppol-registration-pending.png
+   :alt: pending application
+
+It should be automatically activated within a day.
+In the meantime, you can set up the default journal here to receive incoming
+vendor bills.
+
+.. tip::
+  It is also possible to manually trigger the cron that checks the registration status,
+  by going to :menuselection:`Settings --> Technical --> Scheduled Actions --> PEPPOL\: update participant status`
+
+Your application status is going to be updated soon after you are
+registered on the Peppol network.
+
+.. image:: electronic_invoicing/peppol-registration-active.png
+   :alt: active application
+
+All invoices and vendor bills can now be sent directly using the Peppol network.
+
+Partner verification
+--------------------
+
+Before sending an invoice to a partner using the Peppol network, you need to verify
+that they are indeed registered as a Peppol participant as well.
+
+Go to :menuselection:`Accounting --> Customers --> Customers` and open the customer's form.
+Then go to :menuselection:`Accounting tab --> Electronic Invoicing`, select the correct format,
+and make sure their `Peppol EAS` code and the `Endpoint` are filled in.
+Then, click :guilabel:`Verify`. If the partner exists on the network, their `Peppol endpoint validity`
+will be set as Valid.
+
+.. image:: electronic_invoicing/peppol-partner-verify.png
+   :alt: verify partner registration
+
+.. important::
+   While Odoo prefills both the EAS code and the Endpoint number based on the information
+   available for a partner, it is better to confirm directly with the partner,
+   as these are essentially their contact details on the Peppol network.
+
+Send invoices
+-------------
+
+When you are ready to send an invoice via Peppol network, simply click :guilabel:`Send & Print`
+on the invoice form or select multiple invoices in the list view and go to
+:menuselection:`Actions --> Send & Print`.
+
+Make sure that both `BIS Billing 3.0` and `Send via PEPPOL` checkboxes are ticked.
+
+.. image:: electronic_invoicing/peppol-send-print.png
+   :alt: send peppol invoice
+
+The invoices will be queued to be sent in a batch if you send several of them at once.
+Once they are sent via Peppol, the status will be changed to `Processing`.
+The status is changed to `Done` after they have been successfully delivered to the
+partner's Access Point.
+
+.. image:: electronic_invoicing/peppol-message-processing.png
+   :alt: peppol message status
+
+.. tip::
+  By default, the Peppol status column is hidden on the Invoices list view.
+  But you can choose to have it always displayed.
+
+Receive vendor bills
+--------------------
+
+Once a day, a cron checks whether there are any new documents sent to you via the
+Peppol network. These documents will be imported and the corresponding vendor bills
+will be created automatically in draft.
+
+.. image:: electronic_invoicing/peppol-receive-bills.png
+   :alt: peppol receive bills
+
+FAQ
+---
```

Comment:
As this documentation is about general edis, I'd suggest we don't keep this FAQ section. Though if you have a link with general info about peppol registration or a relates subject please insert the link as a seealso.